ContactDetailsApi/V2/Boundary/Request/ServicesoftFetchContactDetailsRequest.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using Microsoft.AspNetCore.Mvc;
+
+namespace ContactDetailsApi.V2.Boundary.Request
+{
+    public class ServicesoftFetchContactDetailsRequest
+    {
+        [FromQuery]
+        public string PaginationToken { get; set; }
+
+        [FromQuery]
+        public int? PageSize { get; set; }
```

Comment:
Should this have a default value?

Reply:
The current default value is `null`. Seems like they're replacing the `null` default with magic number `500` within the gateway.